engine/class_modules/sc_hunter.cpp

```diff
@@ -5344,6 +5384,9 @@ struct wildfire_bomb_t: public hunter_spell_t
 
       am *= 1 + p() -> buffs.flame_infusion -> check_stack_value();
 
+      if ( p() -> options.t28_4pc )
+        am *= 1 + 0.3 + p() -> buffs.mad_bombardier -> check_value();
```

Comment:
According to spell data, this doesn't work this way. The main 2pc spell (https://ptr.wowhead.com/spell=363667/mad-bombardier) is a passive 30% increase. The buff you get from the 4pc (https://ptr.wowhead.com/spell=363805/mad-bombardier) is a separate 50% increase on top of that. Yes, it does mean the final increase is more than 80 - 95.

Also, both spells work off white lists and increase not only base bomb damage but also secondary stuff from wildfire infusion.

Reply:
Oh, you also have to consume the buff after execute. Otherwise it will linger forever.

Reply:
Fixed